DI-1299/compare_b37_and_b38.py

```
@@ -0,0 +1,477 @@
+            project=project_id,
+            name=search_term,
+            name_mode="glob",
+            classname="file",
+            describe={"fields": {"name": True, "archivalState": True}},
+        )
+    )
+
+    return files
+
+
+def get_run_name_from_project_name(b38_project_name):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_run_name_from_project_name`

(ANN201)